### DISCUSSION TITLE
Outdated Node and NPM versions

### DISCUSSION THREAD

**Discussion by CedricBaaklini** [monkeytypegame/monkeytype#5072] [Ideas] [answered]
You guys seem to use an outdated version of both Node and NPM for this project.

<img width="1710" alt="Screenshot 2024-02-16 at 18 33 34" src="https://github.com/monkeytypegame/monkeytype/assets/77132080/3b969c97-e963-4c86-9ff2-7eca080f95fc">

Will you guys ever update your version of Node and NPM?

**Comment by fehmer** [comment, marked as answer]
We still have a year of maintenance left on V18.
![image](https://github.com/monkeytypegame/monkeytype/assets/3728838/13ee6d33-8c8e-492a-9006-84344a70f4e8)
